config_test.go

```diff
@@ -32,7 +32,8 @@ var testConfig = `{
   "PollIntervalSeconds": 60,
   "ServerURL": "mender.io",
   "DeviceID": "1234-ABCD",
-  "ServerCertificate": "/data/server.crt"
+  "ServerCertificate": "/var/lib/mender/server.crt",
```

Comment:
Yup. This does not matter in tests, but it's changed to avoid confusions, where data should be stored.